rfcs/0000-unified-configuration.ftd

```diff
@@ -79,15 +41,11 @@ lang: ftd
 
 \-- fastn.package: my-package
 
-\-- fastn.db-conn my-db:
-db-host: <some-ip>
-db-user: user
-db-password: fastn.env(DB_PASSWORD) \;; suggested syntax
-db-name: user_db
+\-- string my-db-url: $fastn.env(DB_URL)
```

Comment:
I think this will have to be quoted, `$fastn.env(name="DB_URL")`. Checkout how other built in functions behave: https://fastn.com/built-in-functions/.

Reply:
Strings don't need quotes from what I observed. The named argument was needed, thanks!